### PR TITLE
improve prepare data scripts

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

File: app/selenium_ui/conftest.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def jsd_dataset(self):
             self.dataset["requests"] = self.__read_input_file(JSD_DATASET_REQUESTS)
             self.dataset["customers"] = self.__read_input_file(JSD_DATASET_CUSTOMERS)
             self.dataset["agents"] = self.__read_input_file(JSD_DATASET_AGENTS)
-            self.dataset["service_desks"] = self.__read_input_file(JSD_DATASET_SERVICE_DESKS)
+            self.dataset["service_desks"] = self.__read_input_file(JSD_DATASET_SERVICE_DESKS_L)
```
Comment (mmizin) [inline app/selenium_ui/conftest.py:55]: Do we need JSD_DATASET_SERVICE_DESKS_S here?

File: app/util/data_preparation/jsd_prepare_data.py [hunk 2]
```diff
@@ -42,6 +49,7 @@ def wrapper(*args, **kwargs):
             result = func(*args, **kwargs)
             end = timer()
             print(f"{message} finished in {timedelta(seconds=end-start)} seconds")
+            print('------------------------------------------------------')
```
Comment (mmizin) [inline app/util/data_preparation/jsd_prepare_data.py:52]: I have seen that you are removing this print below, maybe you forgot about this print?

**Review by mmizin — APPROVED** [review]

File: app/selenium_ui/conftest.py (hunk 1 above)
Comment (SergeyMoroz0703) [reply app/selenium_ui/conftest.py]: Fixed

File: app/util/data_preparation/jsd_prepare_data.py (hunk 2 above)
Comment (SergeyMoroz0703) [reply app/util/data_preparation/jsd_prepare_data.py:52]: Agree, lets review it after finishing jsd developing as well as PEP8 fixes

**Review by astashys — APPROVED** [review]

**Pull request merged**